src/helpers/index.ts

```diff
@@ -0,0 +1,12 @@
+import { ref } from 'vue'
+import { type SinglePokemonResponse } from '../models/pokedex/index'
+import { useRouter } from 'vue-router'
+
+const router = useRouter()
+export const isFav = ref<SinglePokemonResponse[]>([])
```

Comment:
It's generally not a good idea to define and export reactive variable directly in TS files where they persist as global state. Better to keep state in .vue component files and use props/emits/provide/inject. For reusability, you can create composable functions that return isolated state